Composer/packages/extensions/adaptive-form/src/components/FormRow.tsx

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+import { FieldProps, UIOptions } from '@bfc/extension';
+
+import { resolvePropSchema } from '../utils';
+
+import SchemaField from './SchemaField';
+
+interface FormRowProps extends Omit<FieldProps, 'onChange'> {
+  onChange: (field: string) => (data: any) => void;
+  row: string | [string, string];
+}
+
+const FormRow: React.FC<FormRowProps> = ({
+  id,
+  depth,
+  schema,
+  row,
+  definitions,
+  value,
+  uiOptions,
+  transparentBorder,
+  className,
+  label,
+  rawErrors,
+  onBlur,
+  onFocus,
+  onChange,
+}) => {
+  if (Array.isArray(row)) {
+    return <>Multi field row</>;
+  }
+  const propSchema = resolvePropSchema(schema, row, definitions);
+
+  if (propSchema) {
+    return (
+      <SchemaField
+        key={`${id}.${row}`}
+        className={className}
+        definitions={definitions}
+        depth={depth}
+        id={`${id}.${row}`}
+        label={label === false ? false : undefined}
```

Comment:
Should this just be `undefined`? If it's false, it will hide every child property's label.

Reply:
This is probably just a copy / paste hold over.